<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proj59_guiClass</class>
 <widget class="QMainWindow" name="Proj59_guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proj59_gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="modeAButton">
        <property name="text">
         <string>Mode A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modeBButton">
        <property name="text">
         <string>Mode B</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="evidenceListWidget"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="modesStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>497</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListView" name="myListView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="nextButton">
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveButton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="listButton">
             <property name="text">
              <string>List</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="undoButtonMyList">
             <property name="text">
              <string>Undo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="redoButtonMyList">
             <property name="text">
              <string>Redo</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Page1">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>1</y>
          <width>511</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="idLabel">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="idLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="measurementLabel">
             <property name="text">
              <string>Measurement:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="measurementLineEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="quantityLabel">
             <property name="text">
              <string>Quantity: </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="quantityLineEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="imageClarityLevelLabel">
             <property name="text">
              <string>Image clarity level:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="imageClarityLevelLineEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="photographLabel">
             <property name="text">
              <string>Photograph:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="photographLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateButton">
             <property name="text">
              <string>Update</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="undoButton">
             <property name="text">
              <string>Undo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="redoButton">
             <property name="text">
              <string>Redo</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="myWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="filterShit">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Measurement</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="filterMeasurementLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Quantity</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="filterQuantityLineEdit"/>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="doneButton">
           <property name="text">
            <string>Done</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="filterButton">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Proj59_gui.qrc"/>
 </resources>
 <connections/>
</ui>
